index.js

```diff
@@ -5,14 +5,48 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
+header.style.color = "blue"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+console.log('This is what the traveler object looks like: ', traveler)
+profilediv = document.querySelector('div#profile')
+profilediv.children[1].innerText = traveler.name
+profilediv.children[0].src = traveler.photo
+profilediv.querySelector('em').innerText = traveler.nickname
+profilediv.querySelector('img').alt = traveler.name
```

Comment:
It's best to avoid using `.children` when accessing DOM elements, since it's not as explicit. Your approach using querySelector works well - you can use that same approach for all of this:

```js
profilediv.querySelector('h2').innerText = traveler.name
profilediv.querySelector('em').innerText = traveler.nickname
profilediv.querySelector('img').src = traveler.photo
profilediv.querySelector('img').alt = traveler.name
```